src/stampy_nlp/routes.py

```diff
@@ -0,0 +1,96 @@
+import logging
```

Comment:
I split the main.py file into this one with Flask routes, and a search.py file for function that actually look for stuff. The actual functions haven't been changed